ts/components/services/ServicesTab.tsx

```diff
@@ -0,0 +1,264 @@
+    fontWeight: Platform.OS === "android" ? "normal" : "bold"
+  },
+  textStyle: {
+    fontSize: customVariables.fontSizeSmall
+  },
+  organizationLogo: {
+    marginBottom: 0
+  },
+  icon: {
+    paddingHorizontal: (24 - 17) / 2 // (io-right icon width) - (io-trash icon width)
+  }
+});
+
+class ServicesTab extends React.PureComponent<Props> {
+  /**
+   * For tab Locals
+   */
+  private renderOrganizationLogo = (organizationFiscalCode: string) => {
+    return (
+      <OrganizationLogo
+        logoUri={getLogoForOrganization(organizationFiscalCode)}
+        imageStyle={styles.organizationLogo}
+      />
+    );
+  };
+
+  private organizationContainsText(item: Organization, searchText: string) {
+    return isTextIncludedCaseInsensitive(item.name, searchText);
+  }
+
+  private showChooserAreasOfInterestModal = () => {
+    const {
+      selectableOrganizations,
+      hideModal,
+      selectedOrganizations
+    } = this.props;
+    this.props.showModal(
+      <ChooserListContainer<Organization>
+        items={selectableOrganizations}
+        initialSelectedItemIds={some(new Set(selectedOrganizations || []))}
+        keyExtractor={(item: Organization) => item.fiscalCode}
+        itemTitleExtractor={(item: Organization) => item.name}
+        itemIconComponent={left((fiscalCode: string) =>
+          this.renderOrganizationLogo(fiscalCode)
+        )}
+        onCancel={hideModal}
+        onSave={this.onSaveAreasOfInterest}
+        isRefreshEnabled={false}
+        matchingTextPredicate={this.organizationContainsText}
+        noSearchResultsSourceIcon={require("../../../img/services/icon-no-places.png")}
+        noSearchResultsSubtitle={I18n.t("services.areasOfInterest.searchEmpty")}
+      />
+    );
+  };
+
+  private onSaveAreasOfInterest = (
+    selectedFiscalCodes: Option<Set<string>>
+  ) => {
+    if (this.props.dispatchUpdateOrganizationsOfInterestMetadata) {
+      this.setState({
+        toastErrorMessage: I18n.t(
+          "serviceDetail.onUpdateEnabledChannelsFailure"
+        )
+      });
+      this.props.dispatchUpdateOrganizationsOfInterestMetadata(
+        selectedFiscalCodes
+      );
+      this.props.hideModal();
+    }
+  };
+
+  private renderLocalQuickSectionDeletion = (section: ServicesSectionState) => {
+    const onPressItem = () => {
+      if (this.props.userMetadata && this.props.selectedOrganizations) {
+        const updatedAreasOfInterest = this.props.selectedOrganizations.filter(
+          item => item !== section.organizationFiscalCode
+        );
+        this.setState({
+          toastErrorMessage: I18n.t(
+            "serviceDetail.onUpdateEnabledChannelsFailure"
+          )
+        });
+        this.props.saveSelectedOrganizationItems(
+          this.props.userMetadata,
+          updatedAreasOfInterest
+        );
+      }
+    };
+    return (
+      <TouchableOpacity onPress={onPressItem}>
+        <IconFont
+          name={"io-trash"}
+          color={"#C7D1D9"}
```

Comment:
Avoid declaring styles in line

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1409/commits/97303adcf1dbbde16b4c8a6be32732937f18f978